src/lua/BabblerBomb/Ability.lua

```diff
@@ -0,0 +1 @@
+Script.Load("lua/BabblerBomb/BabblerBombAbility.lua")

```

Comment:
Why don't you directly reference this file as Filehook? Just ends up adding more stuff to the virtual file system without much purpose.